src/tests/ftest/pool/BadQuery.py

```diff
@@ -104,24 +103,24 @@ def test_query(self):
             # setup the DAOS python API
             with open('../../../.build_vars.json') as f:
                 data = json.load(f)
-            CONTEXT = DaosContext(data['PREFIX'] + '/lib/')
+            context = DaosContext(data['PREFIX'] + '/lib/')
 
             # initialize a python pool object then create the underlying
             # daos storage
-            POOL = DaosPool(CONTEXT)
-            POOL.create(createmode, createuid, creategid,
+            pool = DaosPool(context)
+            pool.create(createmode, createuid, creategid,
                         createsize, createsetid, None)
 
-            POOL.connect(connectmode)
+            pool.connect(connectmode)
 
             # trash the pool handle value
             if not handle == 'VALID':
-                POOL.handle = handle
+                pool.handle = handle
 
             if  infoptr == 'NULL':
-                info = POOL.pool_query(InfoPtr=None)
+                info = pool.pool_query(InfoPtr=None)
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'InfoPtr' in method call

(pylint-unused-variable) Unused variable 'info'